text/0000-rfc-process-update.md

```diff
@@ -0,0 +1,166 @@
+---
+tags: ember, ember-cli
+---
+
+* Start Date: 2018-02-04
+* RFC PR: https://github.com/emberjs/rfcs/pull/300
+* Ember Issue: (leave this empty)
+
+# RFC Process Update
+
+## Summary
+
+Introduce the idea of Core Champion to the Ember.js RFC process, merge Ember.js and Ember CLI RFCs repositories,
+and add metadata to make cross-project RFCs easier.
+
+## Motivation
+
+Inspired by Rust, Ember implemented a Request For Commits (RFC) process with the intent to involve the community in discussing the future of the framework.
+The RFC process has proved invaluable.
+Proposals by both Core and community members are discussed and refined,
+with the result coming out much stronger.
+Other projects like React also adopted an RFC process, lending external validation to this idea.
+
+After having been involved with the RFC process for some time, and from feedback from the community, some issues with the current implementation of the process have been noted.
+
+### Confusion about emberjs/rfcs and ember-cli/rfcs
+
+The Ember project currently has two separate RFC processes for Ember.js and Ember CLI.
+
+This leads to confusion due to having to keep track of two different repositories to track progress.
```

Comment:
This leads to confusion because users must keep track of two different repositories in order to monitor progress.

Reply:
How about `people/participants/contributors` over `users`? 😄